client/src/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -6,16 +6,16 @@ exports[`homepage renders homepage 1`] = `
   color: #443023;
 }
 
-.c13:hover {
+.c13 :hover {
```

Comment:
I think there is an issue with this. A space is added between the class and the `:hover`. 
You can see on the homepage a few bugs when hovering a tile, or a button

Reply:
Wow watch a catch ! 👏🏼   I'll investigate 🕵🏼

Reply:
This was due to an issue in the :hover specification for some components.  See https://github.com/refugies-info/karfur/pull/2173/commits/eb8a9ae9fedf08bc89eb466b6eff4073c494a3f3